packages/api/src/webhooks/cannon.ts

```diff
@@ -626,14 +631,24 @@ async function storeResponse(
 export async function resendWebhook(
   webhook: DBWebhook,
   webhookResponse: WebhookResponse
-) {
+): Promise<WebhookResponse> {
   const triggerTime = Date.now();
   const startTime = process.hrtime();
   let resp: Response;
   let responseBody: string;
   let statusCode: number;
   let errorMessage: string;
   try {
+    const sigHeaders = signatureHeaders(
+      webhookResponse.request.body,
+      webhookResponse.sharedSecret,
+      Date.now()
+    );
+    webhookResponse.request.headers = {
+      ...webhookResponse.request.headers,
+      ...sigHeaders,
+    };
```

Comment:
This timestamp actually goes in the payload as well, so we need to update that too
```suggestion
    const timestamp = Date.now()
    const payload = { ...webhookResponse.request.body, timestamp }
    const sigHeaders = signatureHeaders(
      payload,
      webhookResponse.sharedSecret,
      timestamp
    );
    webhookResponse.request.headers = {
      ...webhookResponse.request.headers,
      ...sigHeaders,
    };
```
(then use `payload` instead of `webhookResponse.request.body` below)

Reply:
ah thanks 👍